### PR TITLE
Sil split

### PR DESCRIPTION
**Pull request by madhas8**
- [x] I have read the [CONTRIBUTION GUIDE][contributing]

- [ ] This is a bug

- [x] This is a feature request

- [ ] This is a syntax/style fix

- [ ] This is an improvement to code quality

---
**Please describe your changes below:**
(Note: A clear description of changes is required to approve any pull requests)

_"Changed the implementation for verification because..."_

[contributing]: http://dellemc-symphony.readthedocs.io/en/latest/contributingtosymphony.html

### CODE REVIEW

**Comment by giddyelysium** [comment]
Please add to the CHANGELOG the details of the Changes be it added or changed.

File: common-contract-enricher/pom.xml [hunk 1]
```diff
@@ -9,7 +9,7 @@
     <parent>
         <groupId>com.dell.cpsd</groupId>
         <artifactId>common-messaging-parent</artifactId>
-        <version>2.1.0-SNAPSHOT</version>
+        <version>2.1.1-SNAPSHOT</version>
```
Comment (ganapn) [inline common-contract-enricher/pom.xml]: The last number represents patch/bug fix which is not what we have done

File: common-rabbitmq/src/main/java/com/dell/cpsd/common/rabbitmq/config/ConsumerConfig.java [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+        MessageRecoverer messageRecoverer = new DefaultMessageRecoverer(rabbitTemplate);
+        return new DefaultRetryPolicyAdvice(messageRecoverer, retryPolicy());
+    }
+
+    /**
+     * creates bean for DefaultRetryPolicy
+     * 
+     * @return {@link RetryPolicy}
+     */
+    @Bean
```
Comment (ganapn) [inline common-rabbitmq/src/main/java/com/dell/cpsd/common/rabbitmq/config/ConsumerConfig.java]: Use @Component within the class

File: common-rabbitmq/src/main/java/com/dell/cpsd/common/rabbitmq/config/ConsumerConfig.java [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+    {
+        return new DefaultRetryPolicy();
+    }
+
+    /**
+     * Creates bean for HandlerRegistrar
+     * 
+     * @return {@link HandlerRegistrar}
+     */
+    @Bean
```
Comment (ganapn) [inline common-rabbitmq/src/main/java/com/dell/cpsd/common/rabbitmq/config/ConsumerConfig.java]: Use @Component within the class

File: common-rabbitmq/src/main/java/com/dell/cpsd/common/rabbitmq/config/RabbitConfig.java [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+    {
+        return new RabbitAdmin(rabbitConnectionFactory);
+    }
+
+    /**
+     * create bean for CommonDefaultClassMapper
+     * 
+     * @return {@link CommonDefaultClassMapper}
+     */
+    @Bean
```
Comment (ganapn) [inline common-rabbitmq/src/main/java/com/dell/cpsd/common/rabbitmq/config/RabbitConfig.java]: Use @Component within the class

File: common-testing/pom.xml [hunk 5]
```diff
@@ -9,7 +9,7 @@
     <parent>
         <groupId>com.dell.cpsd</groupId>
         <artifactId>common-messaging-parent</artifactId>
-        <version>2.1.0-SNAPSHOT</version>
+        <version>2.1.1-SNAPSHOT</version>
```
Comment (ganapn) [inline common-testing/pom.xml]: Same comment as above

File: pom.xml [hunk 6]
```diff
@@ -7,7 +7,7 @@
 
     <groupId>com.dell.cpsd</groupId>
     <artifactId>common-messaging-parent</artifactId>
-    <version>2.1.0-SNAPSHOT</version>
+    <version>2.1.1-SNAPSHOT</version>
```
Comment (ganapn) [inline pom.xml]: Same as above

**Comment by ganapn** [comment]
Found conflicts in branch. Please resolve them.

File: common-contract-enricher/pom.xml (hunk 1 above)
Comment (madhas8) [reply common-contract-enricher/pom.xml]: Updated to 2.2.0-SNAPSHOT

File: common-rabbitmq/src/main/java/com/dell/cpsd/common/rabbitmq/config/ConsumerConfig.java (hunk 3 above)
Comment (madhas8) [reply common-rabbitmq/src/main/java/com/dell/cpsd/common/rabbitmq/config/ConsumerConfig.java]: Done

File: common-rabbitmq/src/main/java/com/dell/cpsd/common/rabbitmq/config/ConsumerConfig.java (hunk 2 above)
Comment (madhas8) [reply common-rabbitmq/src/main/java/com/dell/cpsd/common/rabbitmq/config/ConsumerConfig.java]: RetryPolicy is an existing class is common-rabbitmq and is not maintained by SIL as of now. We acn update that class as part of a different story.

File: common-rabbitmq/src/main/java/com/dell/cpsd/common/rabbitmq/config/RabbitConfig.java (hunk 4 above)
Comment (madhas8) [reply common-rabbitmq/src/main/java/com/dell/cpsd/common/rabbitmq/config/RabbitConfig.java]: DefaultClassMapper is part of spring-amqp jar

**Pull request closed without merging**